packages/minimongo/constants.js

```diff
@@ -145,3 +145,5 @@ export const ASYNC_CURSOR_METHODS = [
    */
   'map',
 ];
+
+export const CLIENT_ONLY_METHODS = ["findOne", "insert", "remove", "update", "upsert"];
```

Comment:
Why don't letting backward compatibility on those methods server side with a synchronous version ?

Reply:
It's not possible to use these methods without async without Fibers (aka Meteor 3). You would get back a not resolved Promise.